karpenter/src/schemas/index.tsx

```diff
@@ -0,0 +1,299 @@
+export const KARPENTER_SCHEMAS = [
```

Comment:
Can you please leave a comment here of related documentation or where this is from? If someone needs to update it in the future, it would be good to give them a pointer on where to get started.